### PR TITLE
Scala2.13 + Play2.8

### PR DESCRIPTION
**Pull request by juwen-z**

### CODE REVIEW

File: api/app/actors/EcsBuildActor.scala [hunk 1]
```diff
@@ -118,13 +118,12 @@ class EcsBuildActor @javax.inject.Inject() (
     if (isScaleEnabled) {
       self ! EcsBuildActor.Messages.ConfigureAWS
 
-      system.scheduler.schedule(
+      system.scheduler.scheduleWithFixedDelay(
```
Comment (juwen-z) [inline api/app/actors/EcsBuildActor.scala]: `schedule` is deprecated in `akka 2.6`, using `.scheduleWithFixedDelay` instead. No functional changes.

File: generated/app/FlowGithubV0Client.scala [hunk 2]
```diff
@@ -2093,7 +2093,7 @@ package io.flow.github.v0 {
         requestHeaders: Seq[(String, String)] = Nil
       )(implicit ec: scala.concurrent.ExecutionContext): scala.concurrent.Future[io.flow.github.v0.Response[Unit]] = {
         _executeRequest("DELETE", s"/repos/${play.utils.UriEncoding.encodePathSegment(owner, "UTF-8")}/${play.utils.UriEncoding.encodePathSegment(repo, "UTF-8")}/hooks/${id}", requestHeaders = requestHeaders).map {
-          case r if r.status == 204 => ResponseImpl(body = (), status = r.status, headers = ResponseHeaders(r.headers))
+          case r if r.status == 204 => ResponseImpl(body = (), status = r.status, headers = ResponseHeaders(r.headers.map { case (k, v) => k -> v.toSeq }))
```
Comment (juwen-z) [inline generated/app/FlowGithubV0Client.scala]: The generated code have type mismatch error. I manually resolved.

File: generated/app/FlowGithubV0Client.scala [hunk 3]
```diff
@@ -2445,7 +2445,7 @@ package io.flow.github.v0 {
       ResponseImpl(
         body = parseJson(className, r, f),
         status = r.status,
-        headers = ResponseHeaders(r.headers),
+        headers = ResponseHeaders(r.headers.map { case (k, v) => k -> v.toSeq }),
```
Comment (juwen-z) [inline generated/app/FlowGithubV0Client.scala]: Same as above

**Comment by benwaffle** [comment]
https://github.com/apicollective/apibuilder-generator/pull/589

File: build.sbt [hunk 4]
```diff
@@ -99,13 +97,13 @@ lazy val www = project
     routesGenerator := InjectedRoutesGenerator,
     testOptions += Tests.Argument("-oF"),
     libraryDependencies ++= Seq(
-      "org.webjars" %% "webjars-play" % "2.6.3",
-      "org.webjars" % "bootstrap" % "3.4.1",
+      "org.webjars" % "bootstrap" % "4.4.1-1",
```
Comment (benwaffle) [inline build.sbt]: i don't think `delta` is written for bootstrap 4. need to test

Comment (juwen-z) [reply build.sbt]: I downgraded the version

**Pull request merged**